ni_measurementlink_service/measurement/info.py

```diff
@@ -40,7 +40,6 @@ class ServiceInfo(NamedTuple):
 
     annotations: Dict[str, str] = {}
     """Represents a set of annotations on the service.
-    
```

Comment:
nit: this change is probably not needed

Reply:
Right. [PEP 257](https://peps.python.org/pep-0257/) says

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line.